### PR TITLE
Generalizing plugin

### PR DESCRIPTION
**Pull request by RomanovX**

### CODE REVIEW

File: hooks/addTodayWidgetToProject.js [hunk 1]
```diff
@@ -298,8 +367,7 @@ module.exports = function (context) {
               }
               if (projectContainsSwiftFiles) {
                 buildSettingsObj['SWIFT_VERSION'] = '3.0';
-                buildSettingsObj['ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'] =
-                  'YES';
+                buildSettingsObj['ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES'] = ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES || 'YES';
```
Comment (RomanovX) [inline hooks/addTodayWidgetToProject.js]: You have to pass 'NO'

File: README.md [hunk 2]
```diff
@@ -4,59 +4,71 @@ This plugin extends your existing xcode project by parsing and modifying the pro
...
-│   │   │   my-build-settings.xcconfig
-│   │   │   ...
+### 2. Install the plugin
+* `cordova plugin add https://github.com/DavidStrausz/cordova-plugin-today-widget.git --save`
+* This will not modify anything yet because the hooks only run `after_platform_add`
+* You can add variables to your `config.xml` in order to change some of the settings:
+
+| Variable | Default | Description |
+|-|-|-|
+|WIDGET_PATH| `/www` | Path to the folder that contains your widget folder |
```
Comment (RomanovX) [inline README.md]: fixed

File: README.md [hunk 3]
```diff
@@ -4,59 +4,71 @@ This plugin extends your existing xcode project by parsing and modifying the pro
...
+|WIDGET_NAME| <Name of main project> Widget | Name of your widget |
+|WIDGET_BUNDLE_SUFFIX| widget | The last part of the widget bundle id |
+|ALWAYS_EMBED_SWIFT_STANDARD_LIBRARIES| YES | You might have to turn this off if you use other swift based plugins (such as cordova-plugin-geofence) |
+
+#### Example:
```
Comment (RomanovX) [inline README.md]: fixed

File: hooks/addTodayWidgetToProject.js (hunk 1 above)
Comment (DavidStrausz) [inline hooks/addTodayWidgetToProject.js]: We should add this to the table in the readme, default: 'YES', and in the description: changeable to 'NO' (including the apostrophes)

Comment (RomanovX) [inline hooks/addTodayWidgetToProject.js]: Passing it as a variable value it is automatically converted to string. But I added the 'no' option in the readme

**Pull request merged**